tests/plugins/test_double_plugin.py

```diff
@@ -134,4 +138,29 @@ def test_double_precision(tmpdir, boring_model):
     trainer.test(model)
     trainer.predict(model)
 
-    assert model.training_step == original_training_step
+
+@RunIf(min_gpus=2)
+@pytest.mark.parametrize('ddp_backend', ('ddp', 'ddp_spawn'))
```

Comment:
won't work. 

a) remove ddp
b) add special=True to RunIf

considering that this pr aims at fixing a pickle test, I recommend going for a) or even just merge the test with the above.

Reply:
Done :smiley: